datalad/support/annexrepo.py

```diff
@@ -1357,7 +1357,7 @@ def _get_expected_files(self, files, expr):
 
     @normalize_paths
     def add(self, files, git=None, backend=None, options=None, commit=False,
-            msg=None, dry_run=False,
```

Comment:
FTR `dry_run` used to be used by crawler, but no longer ATM, so I don't think any code is using it any longer

Reply:
Yes. According to a git-grep and the tests, there's no code using it directly. Only the recursive calls to `AnnexRepo.add` were left. And in those cases that call isn't needed.